gtsam/sfm/TranslationFactor.h

```diff
@@ -0,0 +1,83 @@
+/* ----------------------------------------------------------------------------
+
+ * GTSAM Copyright 2010-2020, Georgia Tech Research Corporation,
+ * Atlanta, Georgia 30332-0415
+ * All Rights Reserved
+ * Authors: Frank Dellaert, et al. (see THANKS for the full author list)
+
+ * See LICENSE for the license information
+
+ * -------------------------------------------------------------------------- */
+
+#pragma once
+
+/**
+ * @file TranslationFactor.h
+ * @author Frank Dellaert
+ * @date March 2020
+ * @brief Binary factor for a relative translation direction measurement.
+ */
+
+#include <gtsam/geometry/Point3.h>
+#include <gtsam/geometry/Unit3.h>
+#include <gtsam/linear/NoiseModel.h>
+#include <gtsam/nonlinear/NonlinearFactor.h>
+
+namespace gtsam {
+
+/**
+ * Binary factor for a relative translation direction measurement
+ * w_aZb. The measurement equation is
```

Comment:
Does Z signify 'to' (like a2b)?

Reply:
Not quite. Check out [a discussion in the Google group](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/gtsam-users/erQkDb2LBf0/m4Pz0lWbFQAJ) from a while back for some more details. It's pretty common, in general, for measurement to be styled `z` and the placement of `b` and `w_a` suggests `b` _to_ `w_a` (and not the other way around) 🙂

Reply:
Yes, Z stands for measurement. w_aZb. In this case, it is the translation direction of b from a, but in the world coordinate frame (aligned with the world's coordinate axes, hence the w_).